src/main/java/com/wultra/security/userdatastore/userclaims/UserClaimsService.java

```diff
@@ -15,21 +15,33 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package com.wultra.security.userdatastore;
+package com.wultra.security.userdatastore.userclaims;
 
-import org.junit.jupiter.api.Test;
-import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.Map;
 
 /**
- * Test of spring context start.
+ * Service for user claims.
  *
  * @author Lubos Racansky lubos.racansky@wultra.com
  */
-@SpringBootTest
-class UserDataStoreApplicationTests {
+@Service
+class UserClaimsService {
+
+    // TODO (racansky, 2023-02-17, #4) implement persistence
+    private static final Map<String, Object> data = new HashMap<>();
```

Comment:
Can we use a `LinkedHashMap` to avoid element reordering?

Reply:
This will be replaced in the next PR by JPA.